library/src/scripts/routing/links/backLinkStyles.ts

```diff
@@ -68,6 +69,12 @@ const backLinkClasses = useThemeCache(() => {
         },
     });
 
+    const forHeading = lineHeight => {
```

Comment:
Is the usage of this in an upcoming PR? I was looking for the usage at this point in time and was unable to find it.

The name seems a little vague though. I'm wondering if we should name the function to generate style classes a little differently though. Maybe `headingLinkMixin()` or something like that?